docs/versionPolicy.md

```diff
@@ -0,0 +1,212 @@
+# Versioning
+
+Running `kustomize` means one is running a
+particular version of a program, reading a
+particular version of a [kustomization] file.
+
+## Program versioning
+
+The command `kustomize version` prints a three
+field version tag (e.g. `1.0.11`) that aspires to
+[semantic versioning].
+
+When enough changes have accumulated to
+(subjectively) warrant a new release,
+a [release process] is followed, and the
+fields in the version number are bumped
+per semver.
+
+## Kustomization file versioning
```

Comment:
This section is very useful explaining why doing versioning in `kustomization.yaml` may not be useful at the time being :)

Reply:
thanks